org.eclipse.xtext/src/org/eclipse/xtext/resource/containers/LiveShadowedChunkedContainer.java

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright (c) 2017, 2020 TypeFox GmbH (http://www.typefox.io) and others.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.xtext.resource.containers;
+
+import static org.eclipse.xtext.xbase.lib.IterableExtensions.*;
```

Comment:
If possible, please try to replace the references to the code from xbase.lib. with the Java equivalent.

Reply:
:+1:

Reply:
For "map" there is simple replacement in guava "Iterables". Will replace all others.